### DISCUSSION TITLE
Nominate Anthony Dahanne as a App Monitoring maintainer

### DISCUSSION THREAD

**Discussion by anthonydahanne** [paketo-buildpacks/feedback#296] [App Monitoring Team]
Hey https://github.com/orgs/paketo-buildpacks/teams/app-monitoring-maintainers (there was no discussion space configured for App monitoring), I'd like to propose to promote myself, @anthonydahanne as a App Monitoring maintainer. I believe I proved myself involved with App Monitoring several times.

@paketo-buildpacks/app-monitoring-maintainers Can I get a 👍 / 👎 vote?

Thank you!

**Comment by dmikusa** [comment]
> (there was no discussion space configured for App monitoring)

Oops. Thanks for pointing that out. Fixed.

**Comment by dmikusa** [comment]
We have sufficient votes. This passes. Congratulations @anthonydahanne, welcome to the App Monitoring maintainers team.
